CONTRIBUTING.md

```diff
@@ -176,12 +176,12 @@ flavors for the new Ceph version to the Makefile. At minimum, `ALL_BUILDABLE_FLA
 updated in the Makefile. If distro source is properly configured to support multiple Ceph releases
 and there are no special updates required, they are likely to work with just this minimal change.
 
-Note the `$CEPH_VERSION` variable usually used in `__DOCKERFILE_INSTALL__` is substituted from the
-first field of the flavor name by environment variable substitution.
+Note the `$CEPH_VERSION` and `$CEPH_POINT_RELEASE` variables usually used in `__DOCKERFILE_INSTALL__`
+are extracted from the first field of the flavor name.
 
-In this example, `luminous,x86_64,centos,7,_,centos,7`, `$CEPH_VERSION` will be set to **luminous**.
+In this example, `luminous,centos,7`, `$CEPH_VERSION` will be set to **luminous**.
```

Comment:
Add an example with point release var for readers?

Reply:
There is already one in "### Building with a specific version of Ceph" at README.md
I'd like not duplicating information too much and this part of the documentation doesn't speak about this specific case.